lib/radbeacon/le_scanner.rb

```diff
@@ -1,10 +1,11 @@
 module Radbeacon
 
   class LeScanner
-    attr_accessor :duration
+    attr_accessor :duration, :options
 
     def initialize(duration = 5)
       @duration = duration
+      @options = {}
```

Comment:
Seems like options should be a parameter in the constructor.

Perhaps move `duration` into `options` and that way you could accept them when you are creating the object. Maybe something like this:

```ruby
def initialize(hcitool_duration: 5, mac_filter: nil)
  @hcitool_duration = hcitool_duration
  @mac_filter = mac_filter
end
```